charts/timed/templates/configmap-uwsgi.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "timed.fullname" . }}-uwsgi
+  labels:
+    {{- include "timed.labels" . | nindent 4 }}
+    app.kubernetes.io/component: backend
+data:
+  uwsgi.ini: |
```

Comment:
Why can't this be managed in the container image?

Reply:
laziness
https://github.com/adfinis-sygroup/timed-backend/pull/586